Formula/envoy.rb

```diff
@@ -24,39 +24,35 @@ class Envoy < Formula
   depends_on "automake" => :build
   depends_on "bazelisk" => :build
   depends_on "cmake" => :build
-  depends_on "coreutils" => :build
   depends_on "libtool" => :build
   depends_on "ninja" => :build
   # Starting with 1.21, envoy requires a full Xcode installation, not just
   # command-line tools. See envoyproxy/envoy#16482
   depends_on xcode: :build
   depends_on macos: :catalina
 
-  on_linux do
-    depends_on "gcc@9" => [:build, :test] # Use host/Homebrew GCC runtime libraries.
-    depends_on "python@3.10" => :build
+  uses_from_macos "python" => :build
+
+  on_macos do
+    depends_on "coreutils" => :build
   end
 
   # https://github.com/envoyproxy/envoy/tree/main/bazel#supported-compiler-versions
   fails_with :gcc do
     version "8"
     cause "C++17 support and tcmalloc requirement"
   end
-  # GCC 10 build fails at external/com_google_absl/absl/container/internal/inlined_vector.h:448:5:
-  # error: '<anonymous>.absl::inlined_vector_internal::Storage<char, 128, std::allocator<char> >::data_'
-  # is used uninitialized in this function [-Werror=uninitialized]
```

Comment:
Looks like GCC 10 issue still exists:
```
external/com_google_absl/absl/container/internal/inlined_vector.h:447:5: error: '<unnamed>.absl::InlinedVector<char, 128, std::allocator<char> >::storage_.absl::inlined_vector_internal::Storage<char, 128, std::allocator<char> >::data_' is used uninitialized [-Werror=uninitialized]
    447 |     data_ = other_storage.data_;
        |     ^~~~~
```

May see if there is a commit for newer abseil or try passing in `-Wno-uninitialized`